.github/workflows/container.yml

```diff
@@ -0,0 +1,53 @@
+name: Container
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - '@lagon/serverless@*'
+  pull_request:
+
+env:
+  IMAGE_NAME: serverless
+
+jobs:
+  release-containerimage:
+    name: Create container image
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Build image
+        run: docker build --file docker/Dockerfile --tag $IMAGE_NAME --label "runnumber=${GITHUB_RUN_ID}" .
```

Comment:
Could we use docker buildx to build for both linux/arm64 & linux/amd64?

Reply:
adjusted the workflow to build amd64 and arm64